Doc/library/concurrent.futures.rst

```diff
@@ -194,6 +194,10 @@ And::
       Default value of *max_workers* is changed to
       ``min(32, (os.process_cpu_count() or 1) + 4)``.
 
+   .. versionchanged:: next
```

Comment:
It's not "next" as this was added in a previous release (just find the major.minor version where it was added, I think it's still 3.14)